packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -918,18 +919,21 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             maxOpsSinceLastSummary,
             summarizerClientElectionEnabled,
         );
-        // Create the SummaryManager and mark the initial state
-        this.summaryManager = new SummaryManager(
-            context,
-            this.summarizerClientElection,
-            this.runtimeOptions.summaryOptions.generateSummaries !== false,
-            this.logger,
-            this.runtimeOptions.summaryOptions.initialSummarizerDelayMs,
-        );
-
-        if (this.connected) {
-            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            this.summaryManager.setConnected(this.context.clientId!);
+        // Only create a SummaryManager if summaries are enabled and we are not the summarizer client
+        if (
+            this.runtimeOptions.summaryOptions.generateSummaries !== false
```

Comment:
So, I think this is fine, and definitely want we would prefer long-term. But two things to point out:
1. We lose the telemetry for when an elected summarizer has it disabled: the "SummariesDisabled" event.. We can probably recreate this event using the SummarizerClientElection if we want it. The reason for the event existing: one time a consumer was accidentally disabling summaries, and we couldn't tell from the logs why summaries weren't happening. So we could possibly do without the event, but would need to keep in mind this possibility when troubleshooting.
2. The current implementation of on-demand summaries pass through a SummaryManager, but require it to already be elected & running, so I don't see this as being a problem.

Reply:
For (1) I could move this up to `ContainerRuntime` - it wouldn't fire at the exact point in time that the client fails to deliver a summary but would be enough to raise a red flag in debugging.

Reply:
Moved the telemetry to `ContainerRuntime` in latest.